core/src/main/scala/org/apache/spark/util/SparkUncaughtExceptionHandler.scala

```diff
@@ -36,6 +38,17 @@ private[spark] class SparkUncaughtExceptionHandler(val exitOnUncaughtException:
     val _ = SparkExitCode.OOM
   }
 
+  // The maximum depth to search in the exception cause chain for a fatal error,
+  // as defined by killOnFatalErrorDepth in Executor.scala.
+  //
```

Comment:
Minor: Want to remove Executor.scala as KILL_ON_FATAL_DEPTH is defined in package.scala?